src/routes/configuration.ts

```diff
@@ -1,5 +1,9 @@
 import { Handler } from 'express';
 import { request, RequestOptions } from 'https';
+import { Region, locale } from '../Region';
+
+const region = (process.env.AFTERPAY_REGION as Region) ?? Region.US;
```

Comment:
This can be passed as a parameter. I think it's preferable to read all the env vars in one place and pass them through. 

Checkout is done this way

Reply:
https://github.com/afterpay/sdk-example-server/pull/21 <-- 😬